src/Core_Command.php

```diff
@@ -192,48 +193,50 @@ public function download( $args, $assoc_args ) {
 			WP_CLI::error( 'Skip content is only available for ZIP files.' );
 		}
 
-		$cache = WP_CLI::get_cache();
-		$cache_key = "core/wordpress-{$version}-{$locale}.{$extension}";
-		$cache_file = $cache->has($cache_key);
+		$cache      = WP_CLI::get_cache();
+		$cache_key  = "core/wordpress-{$version}-{$locale}.{$extension}";
+		$cache_file = $cache->has( $cache_key );
 
 		$bad_cache = false;
 		if ( $cache_file ) {
 			WP_CLI::log( "Using cached file '$cache_file'..." );
 			$skip_content_cache_file = $skip_content ? self::strip_content_dir( $cache_file ) : null;
-			try{
+			try {
 				Extractor::extract( $skip_content_cache_file ? $skip_content_cache_file : $cache_file, $download_dir );
 			} catch ( Exception $e ) {
-				WP_CLI::warning( "Extraction failed, downloading a new copy..." );
+				WP_CLI::warning( 'Extraction failed, downloading a new copy...' );
 				$bad_cache = true;
 			}
 		}
 
 		if ( ! $cache_file || $bad_cache ) {
 			// We need to use a temporary file because piping from cURL to tar is flaky
 			// on MinGW (and probably in other environments too).
-			$temp = \WP_CLI\Utils\get_temp_dir() . uniqid('wp_') . ".{$extension}";
-			register_shutdown_function( function () use ( $temp ) {
-				if ( file_exists( $temp ) ) {
-					unlink( $temp );
+			$temp = \WP_CLI\Utils\get_temp_dir() . uniqid( 'wp_' ) . ".{$extension}";
+			register_shutdown_function(
+				function () use ( $temp ) {
+					if ( file_exists( $temp ) ) {
+						unlink( $temp );
+					}
 				}
-			} );
+			);
 
-			$headers = array('Accept' => 'application/json');
+			$headers = array( 'Accept' => 'application/json' );
 			$options = array(
-				'timeout' => 600,  // 10 minutes ought to be enough for everybody
-				'filename' => $temp
+				'timeout'  => 600,  // 10 minutes ought to be enough for everybody
+				'filename' => $temp,
 			);
 
 			$response = Utils\http_request( 'GET', $download_url, null, $headers, $options );
-			if ( 404 == $response->status_code ) {
-				WP_CLI::error( "Release not found. Double-check locale or version." );
-			} else if ( 20 != substr( $response->status_code, 0, 2 ) ) {
+			if ( 404 === (int) $response->status_code ) {
+				WP_CLI::error( 'Release not found. Double-check locale or version.' );
+			} elseif ( 20 !== substr( (int) $response->status_code, 0, 2 ) ) {
```

Comment:
This introduces a bug. `substr()` will return a string, so this strict comparison will always return `true` now.

There are two different ways in which this bug can be fixed:
```suggestion
			} elseif ( '20' !== substr( $response->status_code, 0, 2 ) ) {
```

```suggestion
			} elseif ( 20 !== (int) substr( $response->status_code, 0, 2 ) ) {
```